helper_test.go

```diff
@@ -1161,6 +1161,38 @@ func createVariableSetVariable(t *testing.T, client *Client, vs *VariableSet, op
 	}
 }
 
+func waitForSVOutputs(t *testing.T, client *Client, svID string) {
+	tick := time.NewTicker(2 * time.Second)
+
+	retries := 1
+	maxRetries := 5
+
+	for {
```

Comment:
The solution here is pretty clear, but could it be extracted and reused? terraform-plugin-sdk has a retry function that you can use as inspiration, but essentially I'm thinking that you can provide a function that is retried, and within that you can return an error if len(items) == 0